src/ADAL.PCL/AcquireTokenHandlerBase.cs

```diff
@@ -47,7 +47,8 @@ internal abstract class AcquireTokenHandlerBase
         protected AcquireTokenHandlerBase(RequestData requestData)
         {
             this.Authenticator = requestData.Authenticator;
-            this.CallState = CreateCallState(this.Authenticator.CorrelationId);
+            this.CallState = CreateCallState(this.Authenticator.CorrelationId, requestData.RequestId);
+            requestData.CorrelationId = this.CallState.CorrelationId;
```

Comment:
>requestData.CorrelationId [](start = 12, length = 25)

what is the purpose of putting correlation id in request data? I don't see it being used anywhere.